knowledge_base/Database-Monitoring.md

```diff
@@ -35,43 +44,24 @@ To inspect the definition of a particular object in the schema, use the followin
 To view all foreign keys in the schema, use the following query:
 
 ```SQL
-SELECT conrelid::regclass AS table_name,  
-conname AS foreign_key, 
-pg_get_constraintdef(oid)  
-FROM pg_constraint                                                                                                                                                                                        WHERE  contype = 'f'                                                                                                                                                                                        AND    connamespace = 'public'::regnamespace                                                                                                                                                                
-ORDER  BY conrelid::regclass::text, contype DESC;
+SELECT conrelid::regclass
+AS table_name, conname
+AS foreign_key, pg_get_constraintdef(oid)  
+FROM pg_constraint
+WHERE contype = 'f'
+AND connamespace = 'public'::regnamespace
+ORDER BY conrelid::regclass::text, contype DESC;
 ```
 
 To view all indexes in the schema, use the following query:
 
 ```SQL
 SELECT tablename, indexname, indexdef 
 FROM pg_indexes
-WHERE schemaname = 'public'                                                                                                                                                                                   ORDER BY tablename, indexname;
-```
-
-## Setup Datadog-Heroku Postgres Monitoring
-
-To set up Datadog monitoring for Heroku Postgres, follow the [Heroku Postgres Datadog Setup Guide](https://docs.datadoghq.com/database_monitoring/guide/heroku-postgres/#pagetitle). Additionally, a new Datadog folder with Postgres configuration (`datadog/conf.d/postgres.yaml`) and `prerun.sh` script has been added. This script retrieves the correct credentials for Datadog at runtime.
-
-A helper script is available to complete the setup from a local machine logged into Heroku via CLI. It creates new Datadog user credentials for the DATABASE_URL and creates a new DATADOG schema in Postgres to collect metrics. Use the following command to run the script:
-
-```bash
-pnpm --packages/commonwealth datadog-db-setup commonwealth-staging packages/commonwealth/datadog
+WHERE schemaname = 'public'
+ORDER BY tablename, indexname;
 ```
 
-## Datadog Postgres Dashboards
-
-There are two Dashboards available in Datadog for monitoring Postgres:
-
-### Postgres Metrics Dashboard
```

Comment:
We have Database monitoring enabled now so DB metrics/query information can be found here: https://us5.datadoghq.com/databases/list?dbType=PostgreSQL

Reply:
Moved all this out since redundancies with Datadog.md